web/groups/hlt.json

```diff
@@ -15,29 +21,32 @@
   "BoostedDoubleSVProducer|": "Vertices",
   "BoostedJetONNXJetTagsProducer|": "B tagging",
   "BunchSpacingProducer|": "HLT",
+  "BySiStripClusterMultiplicityEventFilter|LargeSiStripClusterEvents": "AlCa",
+  "CAHitNtupletAlpakaPhase1@alpaka|": "Pixels|GPU",
   "CAHitNtupletAlpakaPhase1@alpaka|*CPUSerial": "Pixels",
   "CAHitNtupletAlpakaPhase1@alpaka|*SerialSync": "Pixels",
-  "CAHitNtupletAlpakaPhase1@alpaka|": "Pixels|GPU",
+  "CAHitNtupletAlpakaPhase2@alpaka|": "Pixels|Alpaka",
```

Comment:
I would prefer that we use `GPU` everywhere instead of mixing `GPU` and `Alpaka`.